src/Doctrine/BaseSingleRelation.php

```diff
@@ -20,7 +20,11 @@ abstract class BaseSingleRelation extends BaseRelation
 
     public function isNullable(): bool
     {
-        return $this->isNullable;
+        if ($this->isNullable) {
+            return $this->isNullable;
+        }
+
+        return false;
```

Comment:
I think (?) this isn't needed. It's true that `isNullable` could be null, but the `:bool` return type will cast null into a `false` boolean.

Reply:
Done.